### PR TITLE
1.0 release blog and website changes

### PR DESCRIPTION
**Pull request by flabbet**
- Updated landing and download page
- New 1.0 release blog post
- Removed shortcuts page (nobody uses version < 0.1.5 anyway)

@Equbuxu @CPKreuz please review blog post, I'm not sure about comparing other apps section, maybe it is unnecessary, tell me what do you think

### CODE REVIEW

File: website/blog/2023-02-27-1.0-release.mdx [hunk 1]
```diff
@@ -0,0 +1,253 @@
...
+click for me, but it's a rather subjective matter. The thing is, people who are used to 
+Photoshop, probably need to learn Aseprite UI and shortcuts for a while, since the layout is a bit different.
+
+We tried solving this by mimicking Photoshop's layout, people are used to it, and finding stuff should be easy. Of course, PixiEditor has its own distinctive style, features, and QoL elements.
+
+But let's check what's really inside PixiEditor 1.0.
+
+# What's new
+
+We've listed some of the new stuff in [previous devlog](/blog/2022/07/04/devlog2), however, I'll describe all of them in short.
```
Comment (Equbuxu) [inline website/blog/2023-02-27-1.0-release.mdx:108]: `listed some of the new stuff .. describe all of them` doesn't make sense

File: website/blog/2023-02-27-1.0-release.mdx [hunk 2]
```diff
@@ -0,0 +1,253 @@
...
+
+Achieving geometrical perfection takes time. But those 2 funky lines can help you with that.
+
+It's like having 4 hands drawing simultaneously.
+
+<video src="/img/blog-posts/1.0/Symmetry.mp4" controls width="700"/>
+
+## New selection transformation
+
+Selection now allows scaling and rotating (with pivot!). Maybe one day we'll add skewing too.
```
Comment (Equbuxu) [inline website/blog/2023-02-27-1.0-release.mdx:134]: we have skewing and even perspective transform lol
try holding down ctrl and dragging one of the corner or side anchors

**Review by Equbuxu — CHANGES_REQUESTED** [review]
"PixiEditor 1.0 is available" text isn't verically centered on smaller screens 
image

I think the roast of inkscape is a bit too long, I'd keep just the part where the line you just drew isn't visible. And remove audacity too, the UI looking old is not objectively a bad thing (and it actually doesn't look old? it used to have a different design which actually looked like it came from win 95, right now it looks fairly modern imo).

File: website/blog/2023-02-27-1.0-release.mdx (hunk 1 above)
Comment (flabbet) [reply website/blog/2023-02-27-1.0-release.mdx:108]: > "PixiEditor 1.0 is available" text isn't verically centered on smaller screens image

It's not a problem imo

File: website/blog/2023-02-27-1.0-release.mdx (hunk 2 above)
Comment (flabbet) [reply website/blog/2023-02-27-1.0-release.mdx:134]: omg I love u

**Pull request merged**